include/platform/configuration_private.h

```diff
@@ -30,4 +33,29 @@
 #define TANK_VALVE_PORT     GPIOB
 #define TANK_VALVE_PIN      GPIO_PIN_6
 
+/************************************************
+ * ALARM CONTROL DEFINITIONS
+ ************************************************/
+
+// GPIO PA9 maps to Pin D8 on the Nucleo-64 board.
+#define ALARM_LED_PORT      GPIOA
+#define ALARM_LED_PIN       GPIO_PIN_9
+
+// GPIO PC7 maps to Pin D9 (PWM) on the Nucleo-64 board.
+
+// Note that timer channels directly map to alternative
+// functions of GPIO pins. TIM3.ch2 maps to PC7 on the 
+// STM32F401RE microprocessor.
+#define ALARM_BUZZ_PORT     GPIOC
+#define ALARM_BUZZ_PIN      GPIO_PIN_7
+#define ALARM_BUZZ_TIM      TIM3
+#define ALARM_BUZZ_CHL      TIM_CHANNEL_2

```

Comment:
This is required to generalize the setup/initialization of the platform so that it is easier to change out the timer in the future (if we need to do that). It probably isn't perfect, but it helps keep all of the configuration information in this file, rather than hardcoding it into `src/platform/init.c`. 

No change made.